examples/js/src/logger.ts

```diff
@@ -0,0 +1,7 @@
+import { Axiom, Logger } from "@axiomhq/js";
+
+const client = new Axiom({ token: process.env.AXIOM_TOKEN || ''});
+
+const logger = new Logger(client, { dataset: process.env.AXIOM_DATASET || '' , autoFlush: true })
+
+logger.info('hello world')
```

Comment:
I think the example should show the usage of `flush()` as the logger pretty much depends on it, probably this example will print nothing.